package.json

```diff
@@ -5,12 +5,19 @@
     "codegen": "graph codegen",
     "build": "graph build",
     "deploy": "graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ salazarguille/zero-collateral",
```

Comment:
This should be updated to since the repo is no longer that address ❓

Reply:
Done